shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/oracle/ddl/OracleDisassociateStatisticsStatement.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.sql.parser.sql.dialect.statement.oracle.ddl;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.apache.shardingsphere.sql.parser.sql.common.segment.ddl.index.IndexSegment;
+import org.apache.shardingsphere.sql.parser.sql.common.segment.dml.column.ColumnSegment;
+import org.apache.shardingsphere.sql.parser.sql.common.segment.generic.table.SimpleTableSegment;
+import org.apache.shardingsphere.sql.parser.sql.common.statement.AbstractSQLStatement;
+import org.apache.shardingsphere.sql.parser.sql.common.statement.ddl.DDLStatement;
+import org.apache.shardingsphere.sql.parser.sql.dialect.statement.oracle.OracleStatement;
+
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * Oracle disassociate statistics statement.
+ */
+@Getter
+@Setter
+@ToString
+public final class OracleDisassociateStatisticsStatement extends AbstractSQLStatement implements DDLStatement, OracleStatement {
```

Comment:
@tristaZero Hi, do we need to concern segments like functions, packages?
As OracleDisassociateStatisticsStatement is DDL Statement,  Below is the definition:
Use the DISASSOCIATE STATISTICS statement to disassociate default statistics or a statistics type from columns, standalone functions, packages, types, domain indexes, or indextypes.
![image](https://user-images.githubusercontent.com/22066046/124371198-e8eb7800-dcb1-11eb-85ec-4403f35d9715.png)
https://docs.oracle.com/en/database/oracle/oracle-database/19/sqlrf/DISASSOCIATE-STATISTICS.html#GUID-6E9A7D93-E28A-469D-97AB-2BECC2EF3C43

Reply:
Hi I think so. Actually, all the segments are supposed to take care of, but I suggest we first extract the commonly used segments, like indexes, types, tables, functions, etc.

Reply:
Thanks for your answer! I'll add them in later PR.

Besides, how do I know which are defined as segments?  I found [this](https://docs.oracle.com/en/database/oracle/oracle-database/19/refrn/DBA_SEGMENTS.html#GUID-7BF4F8C7-1917-4532-B048-C044E17A1754), but function, package etc. are not listed here.

Reply:
To simplify our first stage, you can refer to that page, which looks fine and includes most of the segments we concern about.

Reply:
Ok, thx:)